net/http/authz/store.go

```diff
@@ -94,16 +93,12 @@ func (s *Store) Delete(ctx context.Context, policy string, delete func(*Grant) b
 
 	// We didn't match any grants, don't need to do an update. Return success
 	if len(keep) == len(grantList.Grants) {
-		return nil
+		return sinkdb.Op{}
 	}
 
-	gList := &GrantList{Grants: keep}
-	err = s.sdb.Exec(ctx, sinkdb.Set(key, gList))
-	if err != nil {
-		return errors.Wrap(err)
-	}
-
-	return nil
+	// TODO(tessr): Make this safe for concurrent updates. Will likely require a
```

Comment:
Yes. What if the read-modify-write of Delete interleaves with the read-modify-write of another Delete or a Save?